tests/worker/thread-worker.test.js

```diff
@@ -0,0 +1,9 @@
+const expect = require('expect')
```

Comment:
Thanks @Shinigami92  was really important for me to add a test for the thread worker.
I see also that you removed some code duplications thank you thank you, thread worker was covered but unfortunately nyc does not recognize code covered during other threads run.

Anyway we could add a test for the sendToMainWorker, we will need to mock something I guess

Reply:
Do you want to do this before to merge or do you prefer to handle this later?

Reply:
IMO we should merge it and handle this later

Reply:
k